README.md

```diff
@@ -172,3 +172,35 @@ Example code:
     });
     
     client.terminate();
+    
+Maestro Pololu 
+--------------
+Important notes(Tips and tricks):
```

Comment:
I think that this notes can be extracted from example. What is more, some of them can be resolved, because they look like bugs. If you want, please create appropriate issues and metion @project-capo/developers in the description of issue. README.md should be used to providing information about usage of this library.